### ISSUE TITLE
train file and loss function

### ISSUE THREAD

**Issue by SBaokun** [Beniko95J/MLF-VO#4]
Thank you for your work. Do you have any plans for open source loss functions and training files? Thanks!

**Comment by rginjapan** [comment]
Same question, could you plz give us some updates?

**Comment by Beniko95J** [comment]
Sorry for the late reply. I have uploaded the files for training in the dev branch although they still looks messy... For reference, the codes for training are mainly based on the [Monodepth2](https://github.com/nianticlabs/monodepth2), and we mainly change the architecture of PoseNet and the calculation of training loss. I will find some time to clean up the codes in the dev branch for easy use.
